README.md

```diff
@@ -10,13 +10,15 @@ It is heavily inspired by Brunch.
 
 See [parched-example-app](https://github.com/raisedmedia/parched-example-app), or [parched-tasks-webapp](https://github.com/raisedmedia/parched-tasks-webapp) even. The following will be based on them.
 
+A list of ready made plugins can be found at [parched-plugins](https://github.com/BraveNewWorldDev/parched-plugins)
+
 ## Getting Started
 
 ```bash
 npm install --save gulp parched
 ```
 
-Then update your `Gulpfile.js`
+Then update your `Gulpfile.js`. You can specify an plugins you wish to load here.
```

Comment:
Plugins are specified as dependencies in your `package.json`, not passed to `Parched.setup`